exec/java-exec/src/main/java/org/apache/drill/exec/store/parquet/ParquetRecordWriter.java

```diff
@@ -380,14 +384,21 @@ public void endRecord() throws IOException {
 
       // since ParquetFileWriter will overwrite empty output file (append is not supported)
       // we need to re-apply file permission
-      parquetFileWriter = new ParquetFileWriter(conf, schema, path, ParquetFileWriter.Mode.OVERWRITE);
+      if (useConfiguredBlockSize) {
```

Comment:
The API `ParquetFileWriter(conf, schema, path, ParquetFileWriter.Mode.OVERWRITE)` will cause the Parquet file writer to set the file block size to the greater of the configured files system block size or 128 MB (the ParquetWriter's row group size). 
Drill's Parquet writer will use the block size specified in Drill's options to create a new Parquet row group when the limit is reached (See `ParquetRecodWriter.checkBlockSizeReached()` ). If you set Drill's Parquet block size to the larger of the configured file system block size or 128 MB, you will get the row group to match the HDFS block size. 
Which is what the current code does.
Isn't this what the original JIRA wanted?